src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -343,7 +342,16 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 
 	public unselectActiveCell() {
 		this.model.updateActiveCell(undefined);
-		this.detectChanges();
+	}
+
+	public updateActiveCell(cell: ICellModel) {
+		if (this._model.activeCell !== cell) {
```

Comment:
The model should have this check too - which means you should be able to remove this check.